server/action_creators.js

```diff
@@ -70,6 +71,24 @@ export function addOrder(entries) {
   };
 }
 
+export function changeEntryStatus(orderId, entryIndex, status) {
+  return (dispatch, getState) => {
+    const orders = getState().orders;
+    const orderIndex = orders.findIndex((order) => order.id === orderId);
+
+    return Order.updateEntry(orderId, entryIndex, { status })
+      .then((response) => {
+        const updatedOrders = [
+          ...orders.slice(0, orderIndex),
+          response[0],
+          ...orders.slice(orderIndex + 1)
+        ];
+
+        dispatch(setState({ orders: updatedOrders }));
```

Comment:
In the console, the new state is being loaded, but it's not reflected in the UI.